Add conda-forge installation note (#268)
NeurodataWithoutBorders/nwbinspector: docs/user_guide/installation.rst

    If you are a :matnwb-src:`MatNWB <>` user, please follow the :conda-install:`Conda installation instructions<>` to
    gain access to a Conda/Python terminal and base environment.

.. note::

>     If the `pip` installation pathway is not working on your system, you can also try accessing it from the conda-forge release.

Reviewer: ```suggestion
    You can also install nwbinspector from the conda-forge release.
```
